### PR TITLE
feat(ci): Added Jenkinsfile

### PR DESCRIPTION
**Pull request by rfoster2576**
Added Jenkinsfile to publish both the keycloak operator docker image and Helm Chart to vibrent-ops. Publishing will only occur on a build to the master branch.

AC-94083

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,44 @@
...
+* Support for custom attributes on Realms 
+* Realm update functionality
+* Managing Realms on Unmanaged Keycloak instances
+* Creation of a Helm Chart
+
+### Updating Custom Resource Definitions (CRD)
+Currently the CRDs in the Helm Chart are exact copies of the CRDs found in the /deploy/crds directory. Therefore, when updating a CRD you must replace the appropriate CRD found in /charts/keycloak-operator/crds with the updated CRD to ensure the Helm Chart contains the most current updates when deployed.
+
+### GitHub Actions
+The following three GitHub workflows will run on a PR to Master or a build on Master
```
Comment (RichardMills) [inline README.md]:
```suggestion
Github actions are the primary CI mechanisms used to build and test the keycloak-operator and associated helm charts. However, Github is not capable of publishing to our internal docker and helm repositories, so we also have a Jenkinsfile to handle that. See below.

The following three GitHub workflows will run on a PR to Master or a build on Master
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
+
+1. ci.yml - ensures the tests pass by executing the following makefile commands: test/unit, test/e2e, and test/e2e-local-image
+2. go.yml - ensures the code compiles by executing the following makefile command: code/compile
+3. lint.yml - inspects the code by executing the following makefile commands: setup/linter and code/lint
+
+### Jenkins Pipeline
+The pipeline will lint the Helm Chart anytime it is ran, but will only publish the chart and docker image during a master branch build. Both the chart and docker image will be published to the vibrent-ops project.
```
Comment (RichardMills) [inline README.md:28]:
```suggestion
The Jenkins pipeline is only responsible for publishing the docker container(s) and helm chart for the keycloak operator into our internal docker registry and help chart repository after the github actions perform the other testing activities.

The Jenkins pipeline will lint the Helm Chart anytime it is triggered by a change, but will only publish the chart and docker image during a master branch build. Both the chart and docker image will be published to the vibrent-ops project.
```

**Review by RichardMills — APPROVED** [review]
This looks good. Ship it.

File: README.md (hunk 1 above)
Comment (rfoster2576) [reply README.md]: Made the change

File: README.md (hunk 2 above)
Comment (rfoster2576) [reply README.md:28]: Made the change

**Review by Michael-Valeiras — APPROVED** [review]
LGTM

**Pull request merged**